### PR TITLE
First blogpost

### PR DESCRIPTION
**Pull request by Xe**
Also does this:
- Adds HeroImage component
- Imports it to the MDX context
- Add author metadata for fediverse accounts
- Minor reformatting by my editor's format on save
- Adds Xe author metadata

### CODE REVIEW

File: data/blog/weaponizing-hyperfocus.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+title: 'Weaponizing hyperfocus: Becoming the first DevRel at Tailscale'
+date: '2023-01-25'
+tags:
+  - 'devrel'
+  - 'personal'
+authors: ['xe']
+image: /images/hero/teyvat-utopia.png
```
Comment (tylersmalley) [inline data/blog/weaponizing-hyperfocus.mdx:8]:
```suggestion
```

File: components/ConvSnippet.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return allAuthors[0];
+};
+
+export const ConvSnippet = ({ name, children }: ConvSnippetProps) => {
+  const author = findAuthor(name);
+  const handle = author.name.split(' ', 1)[0];
+
+  return (
+    <div className="flex space-x-4 rounded-lg bg-gray-100 p-4">
+      <div className="flex h-16 w-16 shrink-0 items-center justify-center self-center overflow-hidden rounded-lg bg-gray-200">
```
Comment (tylersmalley) [inline components/ConvSnippet.tsx:26]: Addresses dark theme: 
image


```suggestion
    <div className="flex space-x-4 dark:bg-gray-800 rounded-lg bg-gray-100 p-4">
      <div className="flex h-16 w-16 shrink-0 items-center justify-center self-center overflow-hidden rounded-lg dark:bg-gray-700 bg-gray-200">
```

**Comment by tylersmalley** [comment]
Can we link to all the articles you mentioned?

**Review by tylersmalley — APPROVED** [review]

**Review by WalterHub — APPROVED** [review]
Content looks great.

Noting that Vercel reported a build issue.

**Comment by tylersmalley** [comment]
Vercel can be ignored. It was a test I was running that this PR picked up, which it didn't have support for.

**Pull request merged**